src/js/App/Header/Login.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import { Button } from '@patternfly/react-core/dist/js/components/Button/Button';
 import { login } from '../../jwt/jwt';
+import { PageHeaderTools } from '@patternfly/react-core';
```

Comment:
Shouldn't we load react core directly?
```suggestion
import { PageHeaderTools } from '@patternfly/react-core/dist/js/components/PageHeaderTools/PageHeaderTools';
```

Reply:
Right. No nits not required anymore. Only when we build libraries. Webpack will tree shake correctly here.